tests/unit/view_model_test.py

```diff
@@ -0,0 +1,29 @@
+from todo_app.data.item import Item, ItemStatus
+from todo_app.view_model import ViewModel
+
+item_1 = Item('item_1', 'title_1', ItemStatus.TO_DO)
+item_2 = Item('item_1', 'title_1', ItemStatus.TO_DO)
+item_3 = Item('item_1', 'title_1', ItemStatus.DONE)
+item_4 = Item('item_1', 'title_1', ItemStatus.DOING)
+
+test_items = [item_1, item_2, item_3, item_4]
+
+test_view_model = ViewModel(test_items)
```

Comment:
Would be better to put this all in a fixture so that tests don't inadvertently modify any of it, which would affect other tests.